docs/azure_arc_jumpstart/azure_arc_k8s/day2/calico/_index.md

```diff
@@ -0,0 +1,127 @@
+
+    ![Open Azure Cloud Shell](./03.png)
+
+    Azure CLI:
+
+    ```shell
+    az provider register --namespace 'Microsoft.PolicyInsights'
+    ```
+
+    Azure PowerShell:
+
+    ```powershell
+    Register-AzResourceProvider -ProviderNamespace 'Microsoft.PolicyInsights'
+    ```
+
+    To verify successful registration, run either the below Azure CLI or Azure PowerShell command.
+
+    Azure CLI:
+
+    ```shell
+    az provider show --namespace 'Microsoft.PolicyInsights'
+    ```
+
+    Azure PowerShell:
+
+    ```powershell
+    Get-AzResourceProvider -ProviderNamespace 'Microsoft.PolicyInsights'
+    ```
+
+    ![AzResourceProvider Bash](./04.png)
+
+
+* Azure Policy installed in Arc Connected Cluster
+
+ > by running the the ```kubectl get pods -n gatekeeper-system ``` command, you will notice all pods have been deployed.
+
+![Showing pods deployment](./05.png)
+
+
+## Deploy a customized Azure policy to verify if calicocloud/calico enterprise components enabled in your Azure Arc Kubernetes cluster.
+
+* In the Search bar, look for *Policy* and click on *Definitions* which will show you all of the available Azure policies.
+
+    ![Searching for Azure Policy definitions](./06.png)
+
+    ![Searching for Azure Policy definitions](./07.png)
+
+* Click on *+ Policy definition* to add a new policy. 
+
+    ![Create a Azure policy for Kubernetes cluster](./08.png)
+
+    ![Create a Azure policy for Kubernetes cluster](./09.png)
+
+* In the above example, the *Definition location* of the policy represent the Azure subscription. Also, make sure *Category* is set to *Kubernetes*. We will use `audit` as default effect for this policy. 
+
+  You can either copy the `calicoapipolicy.json` file located in this folder into `POLICY RULE` or create a policy directly with azure cli with below command.
+
+  ```shell
+  az policy definition create --name 'arcdemo-calicoapi-policy' --display-name 'Audit Calico api server' --description 'This policy audit that if calicocloud/calico enterprise components exist' --rules 'calicoapipolicy.json' --mode All
+  ```
+
+* Once the policy is been created, you should be able to find it in *Policy | Definitions* 
+  
+  ![Calico Azure policy ](./10.png) need change
+    
+
+* Click the policy you created, and assign it to your resource group which include the arc clusters. Alternatively, you can assign the policy to entre Azure subscription.
+  
+  ![Assign Calico Azure policy ](./11.png) need change
+  
+
+* After the assignment, the policy task will start the evaluation against arc enabled cluster under your resource group. If you have 2 clusters installed calicocloud/calico enterprise, it will show as  "compliant" with `2 out of 2 ` (The gatekeeper will look for deployment with label 'apiserver' as audit labellist). To check this, go back to the main Policy page in the Azure portal.
+
+    > **Note: The process of evaluation against the cluster can take 30min.**
+
+    ![Azure policy evaluation](./12.png)
```

Comment:
This does not look like the right image as it shows AKS. Also, Resource names can not have a personal reference, they need to be natural.

Reply:
replace another image with gke arc